react/ProductHighlights.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { render } from '@vtex/test-tools/react'
+import { useProduct, ProductContext } from 'vtex.product-context'
+
+import ProductHighlights from './ProductHighlights'
+import ProductHighlightText from './ProductHighlightText'
+import tankTop from './__fixtures__/tankTop'
+
+const mockUseProduct = useProduct as jest.Mock<ProductContext>
+
+mockUseProduct.mockImplementation(() => ({ product: tankTop }))
+
+test("render all collections' highlights", () => {
```

Comment:
Why not use `describe`?

Reply:
I think `test` is leaner

Reply:
But is less organized, isn't?

Reply:
When you have a few tests, no.

Reply:
But when we know that we have to change this? And then when is the time to change? Should we have a pattern?

Reply:
My pattern to create a describe is: there are two tests about the same subject on the same file.